engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-docker.html

```diff
@@ -0,0 +1,31 @@
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="header"></div>
+  <div data-block="selection-summary"></div>
+
+  <div class="well" data-block="filters">
```

Comment:
Hold ove rform before you did the other filtering work so I'll remove for consistency. I think I like the well to set apart the search and filtering and not just the filtering generally speaking.